### ISSUE TITLE
Upgrade Android to 8.0

### ISSUE THREAD

**Issue by hugochien** [mobiscribers/mobiscribers#25]
Upgrade MobiScribe's Android OS from 4.4 to 8.0

**Comment by DanTappan** [comment]
Yes, this is critical. At least to something beyond 4.4. Too many apps won’t run. Tried to install LastPass and it failed- this is going to cripple my use of Dropbox bc I don’t use easy to type (of break) passwords

**Comment by yantar92** [comment]
As I understand, the Android version is limited by hardware. Android 8.0 is likely to be too luggy.

**Comment by webpad** [comment]
> 
> 
> As I understand, the Android version is limited by hardware. Android 8.0 is likely to be too luggy.

Maybe Freescale Android8.0 SDK has something tune-up for MX6SL platform.
